lib/desktop/quicksettings/quick_settings.dart

```diff
@@ -336,91 +324,81 @@ class QuickSettingsState extends State<QuickSettings> {
-                      break;
-                    case "de_DE":
-                      Pangolin.setLocale(context, Locale("id", "ID"));
-                      Pangolin.settingsBox.put("language", "id_ID");
-                      break;
-                    case "id_ID":
-                      Pangolin.setLocale(context, Locale("pl", "PL"));
-                      Pangolin.settingsBox.put("language", "pl_PL");
-                      break;
-                    case "pl_PL":
-                      Pangolin.setLocale(context, Locale("pt", "BR"));
-                      Pangolin.settingsBox.put("language", "pt_BR");
-                      break;
-                    case "pt_BR":
-                      Pangolin.setLocale(context, Locale("ru", "RU"));
-                      Pangolin.settingsBox.put("language", "ru_RU");
-                      break;
-                    case "ru_RU":
-                      Pangolin.setLocale(context, Locale("sv", "SE"));
-                      Pangolin.settingsBox.put("language", "sv_SE");
-                      break;
-                    case "sv_SE":
-                      Pangolin.setLocale(context, Locale("uk", "UA"));
-                      Pangolin.settingsBox.put("language", "uk_UA");
-                      break;
-                    case "uk_UA":
-                    default:
-                      Pangolin.setLocale(context, Locale("en", "US"));
-                      Pangolin.settingsBox.put("language", "en_US");
-                      break;
-                  }
-                }),
-              ])),
+          child:
+              GridView.count(physics: BouncingScrollPhysics(), crossAxisCount: 4, childAspectRatio: 3 / 4, children: [
+            buildTile(Icons.network_wifi, LocaleStrings.pangolin.qsWifi, () {
+              setState(() {
+                HiveManager.set("wifi", false);
+              });
+              Provider.of<WindowsData>(context, listen: false).add(child: WirelessApp(), color: Colors.deepOrange[700]);
+              hideOverlays();
+            }),
+            buildTile(Icons.palette, LocaleStrings.pangolin.qsTheme, changeColor),
+            buildTile(Icons.battery_full, '85%', changeColor),
+            buildTile(Icons.do_not_disturb_off, LocaleStrings.pangolin.qsDnd, changeColor),
+            buildTile(Icons.lightbulb_outline, LocaleStrings.pangolin.qsFlashlight, changeColor),
+            buildTile(Icons.screen_lock_rotation, LocaleStrings.pangolin.qsAutorotate, changeColor),
+            buildTile(Icons.bluetooth, LocaleStrings.pangolin.qsBluetooth, changeColor),
+            buildTile(Icons.airplanemode_inactive, LocaleStrings.pangolin.qsAirplanemode, changeColor),
+            buildTile(Icons.invert_colors_off, LocaleStrings.pangolin.qsInvertcolors, changeColor),
+            buildTile(Icons.language, LocaleStrings.pangolin.qsChangelanguage, () {
+              switch (Localizations.localeOf(context).toString()) {
```

Comment:
this seems pretty janky, i'd just check current locale, get index from supported locales and set current locale to index + 1

Reply:
I got, instead of switch case. 
😓 I call with it was, so I'll fix as you said